functionary/ui/urls.py

```diff
@@ -1,41 +1,63 @@
 from django.conf import settings
 from django.conf.urls.static import static
-from django.urls import path
+from django.contrib.auth.decorators import login_required
+from django.urls import include, path
 
 from .views import environments, functions, home, packages, tasks, teams
 
 app_name = "ui"
 
 urlpatterns = [
-    path("", home.home, name="home"),
+    path("", login_required(home.home), name="home"),
```

Comment:
This obviously works, but it's slightly confusing to see something clearly labeled as a decorator used this way.  For class based views, it looks like the thing you want is [`LoginRequiredMixin`](https://docs.djangoproject.com/en/4.1/topics/auth/default/#the-loginrequired-mixin), unless there was a specific reason you went this way with it.

Reply:
That's a better way :D